open_city_profile/tests/snapshots/snap_test_graphql_api_schema.py

```diff
@@ -727,6 +741,24 @@
   clientMutationId: String
 }
 
+input VerifiedPersonalInformationAddressInput {
+  streetName: String
+  postalCode: String
+  city: String
```

Comment:
I used the same terminology that is already used in [Address](https://github.com/City-of-Helsinki/open-city-profile/blob/develop/profiles/models.py#L310). I agree it's not the best term, but I used it for consistency.

Then again, for the national identification number, I didn't use `ssn` which is already used in [SensitiveData](https://github.com/City-of-Helsinki/open-city-profile/blob/develop/profiles/models.py#L250). So I guess I wasn't that consistent after all 🤔 

I think I'm fine with not being consistent with poorly chosen names.

Reply:
Renamed `city` to `post office`.

Also renamed `street name` to `street address` because that field, in addition to street name, may also contain e.g. house number, apartment number and such. All of those together is called `street address`.